website/content/en/docs/helm/quickstart.md

```diff
@@ -11,32 +11,42 @@ This guide walks through an example of building a simple nginx-operator powered
 
 ```sh
-operator-sdk new nginx-operator --api-version=example.com/v1alpha1 --kind=Nginx --type=helm
-cd nginx-operator
+$ mkdir nginx-operator
+$ cd nginx-operator
+$ operator-sdk init --plugins=helm.operator-sdk.io/v1 --domain=com --group=example --version=v1alpha1 --kind=Nginx
 ```
 
 This creates the nginx-operator project specifically for watching the
 Nginx resource with APIVersion `example.com/v1alpha1` and Kind
 `Nginx`.
 
-For Helm-based projects, `operator-sdk new` also generates the RBAC rules
-in `deploy/role.yaml` based on the resources that would be deployed by the
+For Helm-based projects, `operator-sdk init` also generates the RBAC rules
+in `config/rbac/role.yaml` based on the resources that would be deployed by the
 chart's default manifest. Be sure to double check that the rules generated
-in `deploy/role.yaml` meet the operator's permission requirements.
+in `config/rbac/role.yaml` meet the operator's permission requirements.
 
 To learn more about the project directory structure, see the
 [project layout][layout-doc] doc.
 
 ### Use an existing chart
 
-Instead of creating your project with a boilerplate Helm chart, you can also use `--helm-chart`, `--helm-chart-repo`, and `--helm-chart-version` to use an existing chart, either from your local filesystem or a remote chart repository.
+Note that, instead of creating your project with a boilerplate Helm chart, you can also use `--helm-chart`, `--helm-chart-repo`, and `--helm-chart-version` to use an existing chart, either from your local filesystem or a remote chart repository. To know more about Helm charts see its [docs](https://helm.sh/docs/topics/charts/).
 
-If `--helm-chart` is specified, `--api-version` and `--kind` become optional. If left unset, the SDK will default `--api-version` to `charts.helm.k8s.io/v1alpha1` and will deduce `--kind` from the specified chart.
+If `--helm-chart` is specified, the API flags (`--domain`, `--group`, `version`, `kind`) become optional. If left unset, the SDK will default will be:
 
-If `--helm-chart` is a local chart archive or directory, it will be validated and unpacked or copied into the project.
+| Flag | Value |
+| :--- | :---    |
+| domain | my.domain |
+| group |  helm.operator-sdk |
+| kind |  deduce from the specified chart |
+| version | v1alpha1 |
+
+If `--helm-chart` is a local chart archive (e.g `example-chart-1.2.0.tgz`) or directory , it will be validated and unpacked or copied into the project.
 
 Otherwise, the SDK will attempt to fetch the specified helm chart from a remote repository.
 
+**Note** For example, in order to  be allowed to built a project with the flag `--helm-chart=stable/memcached` from `https://github.com/helm/charts/tree/master/stable/memcached` is required to have the `stable` repository added. You can check the list of repos added locally with ` helm repo --list`. For further information see: [The Chart Repository Guide](https://helm.sh/docs/topics/chart_repository/).  
```

Comment:
Remove this since it is covered in line 52-54. At the end of that line, we can add "Use `helm repo add` to configure repositories."

```suggestion
```

Reply:
I changed as suggested. I am trying to cover a question that is common when the dev did not come from Helm but is trying to use it.